templates/bake/Model/enum.twig

```diff
@@ -24,11 +24,15 @@
 {{ DocBlock.classDescription(name, 'Enum', [])|raw }}
 enum {{ name }}: {{ backingType }} implements EnumLabelInterface
 {
+{% if cases %}
+    {{ Bake.concat('\n    ', cases) }}
+
+{% endif %}
     /**
      * @return string
      */
     public function label(): string
     {
-        return Inflector::humanize(Inflector::underscore($this->name));
+        return Inflector::humanize(mb_strtolower($this->name));
```

Comment:
Why this change again?

Reply:
Well, because it doesnt work otherwise with UPPER_CASE, as it would become U P P E R C A S E for display field in form. See my other comment and referenced core code.

Reply:
reverted now